### PR TITLE
New package : pgadmin3 1.20.0

### PR DESCRIPTION
**Pull request by arete**
Signed-off-by: Gerardo Di Iorio <arete74@gmail.com>

### CODE REVIEW

File: srcpkgs/pgadmin3/template [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Template file for 'pgadmin3'
+pkgname=pgadmin3
+version=1.20.0
+revision=1
+only_for_archs="i686 x86_64"
```
Comment (xtraeme) [inline srcpkgs/pgadmin3/template]: This `only_for_archs` is bogus, remove it.

**Comment by arete** [comment]
ok .i have update the commit

**Comment by pullmoll** [comment]
To me the license seems to be `BSD` style!? In any case you should add
```
post_install() {
        vlicense LICENSE
}

**Comment by arete** [comment]
Ok add license file

**Comment by xtraeme** [comment]
`Postgresql license` is not a valid license, set it to `BSD`.

**Comment by arete** [comment]
ok set license to BSD

**Pull request merged**